<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QDialog" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>account</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>591</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Account</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>info</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_nom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_prenom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>date de naissanse:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Role:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="ajout_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_cin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero telephone:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_num">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse email:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_email">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ajout_role">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>341</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
